test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -986,6 +986,10 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 		} );
 
 		it( 'Tracks "wpcom_block_editor_convert_to_template_part"', async function () {
+			// Reload editor to start from consistent clean slate for tests.
+			await this.driver.navigate().refresh();
```

Comment:
I'm not super familiar with these tests, but would this kind of problem effect other tests too? I'm curious if it should be something that happens generally before most tests

Reply:
It can effect other tests as well! There are some other places in this file where we trigger a refresh to avoid inconsistencies with scrolling and clicking items (like the floating help button intercepting block clicks, etc.). Currently, those all happen in tests that run later down the chain than this one. The general tests that run before this one insert a lot of blocks and patterns into the page so it was starting at a pretty messy slate.

Most tests could probably benefit from refreshing first. The trade-off is the reload times adding up and making the e2e tests take longer to run (which I was told was a concern when we had previously added some wait timers), which is why it may only make sense to add these refreshed where necessary.

Reply:
Fair enough! That makes sense to me.